public/menu.json

```diff
@@ -0,0 +1,26 @@
+[
+  {
```

Comment:
this should be wrapped in a products key

Reply:
Fixed this.